src/payments-welcome/exit-survey-modal.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	Button,
+	Modal,
+	CheckboxControl,
+	TextareaControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import strings from './strings';
+import wcpayTracks from './tracks';
+
+/**
+ * Provides a modal requesting customer feedback.
+ *
+ */
+function ExitSurveyModal(): JSX.Element | null {
+	const [ isOpen, setOpen ] = useState( true );
+	const [ isHappyChecked, setHappyChecked ] = useState(false);
+	const [ isInstallChecked, setInstallChecked ] = useState(false);
+	const [ isMoreInfoChecked, setMoreInfoChecked ] = useState(false);
+	const [ isAnotherTimeChecked, setAnotherTimeChecked ] = useState(false);
+	const [ isSomethingElseChecked, setSomethingElseChecked ] = useState(false);
+	const [ comments, setComments ] = useState( '' );
+	
+	const closeModal = () => setOpen( false );
```

Comment:
I think we need to set [wc_calypso_bridge_payments_dismissed](https://github.com/Automattic/wc-calypso-bridge/pull/703/files#diff-6913cd5881364970fe5a0344296a03005ccab72b20b844d5e040e649ba91531aR184) to `yes` here

Reply:
Ah sorry I should have mentioned this in the description. There's a separate issue for this https://github.com/Automattic/wc-calypso-bridge/issues/691, so this PR is just handling the survey.

Reply:
Thank you for the clarification 👍